setup.py

```diff
@@ -15,12 +15,12 @@
     sys.exit(1)
 
 install_requires = [
-    'sphinxcontrib-applehelp',
-    'sphinxcontrib-devhelp',
-    'sphinxcontrib-jsmath',
-    'sphinxcontrib-htmlhelp',
-    'sphinxcontrib-serializinghtml',
-    'sphinxcontrib-qthelp',
+    'sphinxcontrib-applehelp>=1.0.2',
+    'sphinxcontrib-devhelp>=1.0.2',
+    'sphinxcontrib-jsmath>=1.0.1',
+    'sphinxcontrib-htmlhelp>=2.0.0',
+    'sphinxcontrib-serializinghtml>=1.1.5',
```

Comment:
IMO, it's enough to pin only htmlhelp and serializinghtml.

Reply:
It seems like we should pin them all, so that we don't hit this issue in the future? I imagine there will also be breaking changes over time, and we want users to have the latest version. I don't feel strongly though.

Reply:
-1: too strict